README.md

```diff
@@ -1,13 +1,97 @@
 # DESMO-SDK
+[![License: MIT](https://img.shields.io/badge/License-MIT-yellow.svg)](https://opensource.org/licenses/MIT)
+![](https://img.shields.io/badge/eslint-3A33D1?style=flat-square&logo=eslint&logoColor=white)
+![](https://img.shields.io/badge/prettier-1A2C34?style=flat-square&logo=prettier&logoColor=F7BA3E)
 
 **Overview**
-> In the context of the [DESMO-LD](https://github.com/vaimee/desmo/tree/c763cec12f6c9060a9f1a3335ff4cff60ece3df2) an  important part of the project relay on storing The Thing Description Directories (TDDs) URLs on-chain so that they can be selected to provide the query result and increase its score, thus the DESMO-SDK is a tool that provides all the features required to interact with then.
+> In the context of the [DESMO-LD](https://github.com/vaimee/desmo/tree/c763cec12f6c9060a9f1a3335ff4cff60ece3df2) an  important part of the project relay on storing The Thing Description Directories (TDDs) URLs on-chain so that they can be selected to provide the query result and increase its score, thus the DESMO-SDK is a tool that provides all the features required to interact with them.
 
+- [DESMO-SDK](#desmo-sdk)
+  - [Features](#features)
+  - [Installation](#installation)
+  - [Usage Examples](#usage-examples)
+    - [DesmoHub contract](#desmohub-contract)
+      - [Register your TDD](#register-your-tdd)
+      - [Get the user's TDD](#get-the-users-tdd)
+    - [Desmo Contract](#desmo-contract)
+      - [Buy a query](#buy-a-query)
+  - [Documentation](#documentation)
+  - [Acknowledgments](#acknowledgments)
+  - [Contact](#contact)
```

Comment:
Missing the collapse listing as we have in https://github.com/vaimee/zion#zion---a-scalable-thing-description-directory